src/pages/chat/Chat.tsx

```diff
@@ -141,6 +153,88 @@ export default function Chat() {
   const [previousPath, setPreviousPath] = useState<string>("");
   const navigate = useKeepQueryParamsNavigate();
   const { isXXS, isXS, isS } = useBreakpoints();
+  const { showModal, modalTypes } = useModal();
+  const { bosonXmtp } = useChatContext();
+  const notifiConfig = getNotifiConfig();
+
+  // TODO: ideally if notifiConfig is null, do not call useNotifiClient()
```

Comment:
there is no way to enable/disable this hook AFAIK
```suggestion
```

Reply:
ok